### PR TITLE
Fix ci image versions

### PR DESCRIPTION
**Pull request by suzhoum**
Issue #, if available:

Description of changes:
This PR fixes a few bugs in CI.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/autogluon/cloud/backend/ray_backend.py [hunk 1]
```diff
@@ -395,6 +395,10 @@ def _get_image_uri(self, framework_version: str, instance_type: str, custom_imag
                 image_scope="training",
                 instance_type=instance_type,
             )
+            if instance_type == "gpu":
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply src/autogluon/cloud/backend/ray_backend.py:398]: This is just for testing, I created a few images with the correct ray version. I can remove them before merging.

File: src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml [hunk 2]
```diff
@@ -44,3 +44,4 @@ head_node_type: head
 initialization_commands:
   # To get access to DLC image
   - aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
+  - aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-west-2.amazonaws.com
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml:47]: If user uses images before `0.8.0`, I think they are hosted in `us-east-1`

File: tests/conftest.py [hunk 3]
```diff
@@ -10,10 +10,10 @@
 
 
 class CloudTestHelper:
-    cpu_training_image = "369469875935.dkr.ecr.us-east-1.amazonaws.com/autogluon-nightly-training:cpu-latest"
-    gpu_training_image = "369469875935.dkr.ecr.us-east-1.amazonaws.com/autogluon-nightly-training:gpu-latest"
-    cpu_inference_image = "369469875935.dkr.ecr.us-east-1.amazonaws.com/autogluon-nightly-inference:cpu-latest"
-    gpu_inference_image = "369469875935.dkr.ecr.us-east-1.amazonaws.com/autogluon-nightly-inference:gpu-latest"
+    cpu_training_image = "369469875935.dkr.ecr.us-east-1.amazonaws.com/weisu:cpu-latest"
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply tests/conftest.py:13]: Sorry this is for testing. I created those just now.

File: src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tonyhoo) [reply src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml:47]: If that's the case, shall we host all images in the same region say "us-east-1"?

File: src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml [hunk 4]
```diff
@@ -6,7 +6,7 @@ max_workers: 1
 
 docker:
     # The image uri will be dynamically replaced by cloud predictor
-    image: "763104351884.dkr.ecr.us-east-1.amazonaws.com/autogluon-training:0.7.0-cpu-py39-ubuntu20.04"
+    image: "763104351884.dkr.ecr.us-west-2.amazonaws.com/autogluon-training:1.1.0-cpu-py310-ubuntu20.04"
```
[the remark this answers is not in the document]
Comment (prateekdesai04) [reply src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml:9]: I believe the image is already present in east-1

File: src/autogluon/cloud/backend/ray_backend.py [hunk 5]
```diff
@@ -395,6 +395,10 @@ def _get_image_uri(self, framework_version: str, instance_type: str, custom_imag
                 image_scope="training",
                 instance_type=instance_type,
             )
+            if "g4dn" in instance_type:
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply src/autogluon/cloud/backend/ray_backend.py:398]: I will actually remove the whole commit 6cc23a75c95c94602558526c0e2728fbee951fa7. This is to override the logic of getting the versioned DLC images using the sagemaker api.

File: src/autogluon/cloud/utils/ag_sagemaker.py [hunk 6]
```diff
@@ -39,6 +39,10 @@ def __init__(
                 image_scope="training",
                 instance_type=instance_type,
             )
+            if "g4dn" in instance_type:
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply src/autogluon/cloud/utils/ag_sagemaker.py:42]: same here, I will drop the commit.

File: src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (suzhoum) [reply src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml]: Updated

File: tests/unittests/cluster/test_distributed_training.py [hunk 7]
```diff
@@ -44,9 +44,11 @@ def test_distributed_training(test_helper, framework_version):
             predictor_init_args=predictor_init_args,
             predictor_fit_args=predictor_fit_args,
             custom_image_uri=image_uri,
+            framework_version=framework_version,
             backend_kwargs={
                 "initialization_commands": [
                     "aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com",
+                    "aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-west-2.amazonaws.com",
```
[the remark this answers is not in the document]
Comment (suzhoum) [reply tests/unittests/cluster/test_distributed_training.py]: Removed since they are now available in `us-east-1`

**Review by prateekdesai04 — APPROVED** [review]
LGTM!

**Pull request merged**